index.html

```diff
@@ -96,9 +110,9 @@ <h2 class="project-title">Profesional Art Printing Data</h2>
               <li><a href="">Ruby</a></li>
             </ul>
           </div>
-          <a href="#" class="see-project-btn">See project</a>
-          <div class="project-box">
-            <h2 class="project-title">Profesional Art Printing Data</h2>
+          <!-- <a href="#" class="see-project-btn">See project</a> -->
```

Comment:
- [ ] please align the different language and skills to the left like in the figma
yours
![Screenshot (405)_LI](https://user-images.githubusercontent.com/65298048/188020975-d6e9b189-52a0-41f5-91ac-489f184849b3.jpg)
figma
![Screenshot (404)](https://user-images.githubusercontent.com/65298048/188021057-fc743686-6377-48f8-9051-1e51e01f32ec.png)

Reply:
I have aligned the texts to the right thank you for the sug
![alignitems](https://user-images.githubusercontent.com/96125730/188123561-1f48fd3f-4d91-4ec7-a7ba-62cb581742e0.png)
gestion